app/lib/AccountRequirement.scala

```diff
@@ -22,7 +22,8 @@ import Implicits._
 
 object AccountRequirements {
 
-  val All = Seq(FullNameRequirement, TwoFactorAuthRequirement, SponsorRequirement)
+//  val All = Seq(FullNameRequirement, TwoFactorAuthRequirement, SponsorRequirement)
```

Comment:
Can we remove the commented out line?

Reply:
@necrophonic See https://github.com/ONSdigital/gu-who/pull/1/commits/a05231505c0b98b5487cdecfbb85e4faa323423d